platform/security/common/src/main/java/ddf/security/assertion/impl/SecurityAssertionImpl.java

```diff
@@ -204,6 +204,9 @@ private void parseToken(SecurityToken securityToken) {
                     if (AuthnStatement.AUTHN_INSTANT_ATTRIB_NAME.equals(name)) {
                       authenticationStatement.setAuthnInstant(DateTime.parse(value));
                     }
+                    if (AuthnStatement.SESSION_INDEX_ATTRIB_NAME.equals(name)) {
+                      authenticationStatement.setSessionIndex(value);
+                    }
```

Comment:
This is where the SecuityAssertion copies the session index out of the SecurityToken